config/default.yaml

```diff
@@ -763,6 +763,9 @@ instance:
   securitytxt: |
     Contact: https://github.com/Chocobozzz/PeerTube/blob/develop/SECURITY.md
     Expires: 2025-12-31T11:00:00.000Z'
+  logs:
```

Comment:
You already have a log section: https://github.com/Chocobozzz/PeerTube/blob/develop/config/default.yaml#L230

Please also add this block to `production.yaml.example`